### PR TITLE
ISPN-11782 xsite monitoring and troubleshooting

### PR DESCRIPTION
**Pull request by oraNod**
https://issues.redhat.com/browse/ISPN-11782

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/ref_logs_xsite.adoc [hunk 1]
```diff
@@ -0,0 +1,65 @@
+[id='xsite_logs-{context}']
+= Cross-Site Log Messages
+{brandname} prints different log messages related to cross-site replication.
```
Comment (oraNod) [inline documentation/src/main/asciidoc/topics/ref_logs_xsite.adoc:3]: @pruivo hey I think there are other trace messages that we might want to add here. For example "IRAC operation failed!" but I couldn't find any identifier (ISPN0000) and I'm not quite sure about them. Do you think we should add those messages here?

File: documentation/src/main/asciidoc/topics/proc_troubleshooting_xsite.adoc [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+. Open a terminal in `{server_home}`.
+. Create a {brandname} CLI connection.
+. Adjust run-time logging levels to capture DEBUG or TRACE messages if necessary.
++
+For example, the following command enables TRACE log messages for the org.infinispan.XSITE category:
++
+[source,options="nowrap",subs=attributes+]
+----
+[//containers/default]> logging set --level=TRACE org.infinispan.XSITE
```
Comment (pruivo) [inline documentation/src/main/asciidoc/topics/proc_troubleshooting_xsite.adoc:17]: we don't allow TRACE in runtime (without restarting the server). You can enable DEBUG.

Comment (oraNod) [reply documentation/src/main/asciidoc/topics/proc_troubleshooting_xsite.adoc]: updated

**Comment by pruivo** [comment]
integrated! thanks @oraNod !

**Pull request merged**